backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/internalbusinesslogic/ShipmentData.java

```diff
@@ -79,11 +70,7 @@ private ShipmentData(
 
 	public StockQtyAndUOMQty computeInvoicableQtyDelivered(@NonNull final InvoicableQtyBasedOn invoicableQtyBasedOn)
 	{
-		if(empty)
-		{
-			return StockQtyAndUOMQtys.createZero(productId, getQtyNominal().getUomId());
-		}
-		Quantity deliveredInUom;
+		final Quantity deliveredInUom;
 		switch (invoicableQtyBasedOn)
```

Comment:
hi @metas-ts , the changes cover 2 scenarios:
1. removing this line is what allows our scenario to fallback to default QtyDelivered/QtyToInvoice (meaning existing values from C_Invoice_Candidate record) => we want to fallback to default if no ICIOLs are available
2. loading all ICIOLs prevents falling back to default values if there are any iciols out there (basically covers the reversed & reopen scenarios)